content-src/components/DiscoveryStreamComponents/DSContextFooter/_DSContextFooter.scss

```diff
@@ -0,0 +1,109 @@
+$status-green: #058B00;
+$status-dark-green: #7C6;
+
+.story-footer {
+  color: var(--newtab-text-secondary-color);
+  display: flex;
+  height: 24px;
```

Comment:
Can we get rid of height, this is keeping extra 24px blank space when we don't have sponsored context or messaging

Reply:
Also, I think its safe to remove display: flex and align-items: center as we are making status-message flex container